src/style.css

```diff
@@ -173,6 +173,18 @@
   animation: dash 1.5s ease-in-out infinite;
 }
 
+.rmsc__arrow-svg {
+  fill: none;
+  stroke: currentColor;
+  stroke-width: 2;
+}
+
+.rmsc__cross-svg {
+  fill: none;
+  stroke: currentColor;
+  stroke-width: 2;
+}
+
```

Comment:
style SVG element using CSS value instead of writing hardcoded code. now, the user could override this value if he wants.